### ISSUE TITLE
Email login no longer available (international) - "token" solution works

### ISSUE THREAD

**Issue by NateRobinsonS** [pnbruckner/ha-life360#40]
My account suddenly stopped working a couple of days ago. Re-entering the credentials (email and password) didn't work.

Deleting the old "account" and **using the "Token" method worked** (even though I have no phone number attached to the account).

I'm not in the USA, and Life360's web page seemed to recognize this. The server may be configured to handle access from outside the US differently.

Posting this to support others. Feel free to close the issue whenever it feels right.

Using version 0.5.4 of the integration on HA 2024.11.3.

I really appreciate the effort put into keeping this going.

**Comment by jasonwragg** [comment]
> My account suddenly stopped working a couple of days ago. Re-entering the credentials (email and password) didn't work.
> 
> Deleting the old "account" and **using the "Token" method worked** (even though I have no phone number attached to the account).
> 
> I'm not in the USA, and Life360's web page seemed to recognize this. The server may be configured to handle access from outside the US differently.
> 
> Posting this to support others. Feel free to close the issue whenever it feels right.
> 
> Using version 0.5.4 of the integration on HA 2024.11.3.
> 
> I really appreciate the effort put into keeping this going.

Out of interest how did you use the token method if you can't sign in the life360 website?

**Comment by NateRobinsonS** [comment]
Life360 seems to have changed the login method. Rather than using a password (which was previously set), a challenge was sent to my email address. This allowed me to log in to their site.

I suspect that the email+password combination does not work any longer.

**Comment by ctaggmk** [comment]
email + password worked for me just now. I did have to reset password from inside mobile app first but then all good

**Comment by sbreeves1984** [comment]
I’ve tried to reset my password but HA comes up with invalid authentication. I tried the token method and it failed to initialise Life360 in HA. 
any suggestions?

**Comment by sbreeves1984** [comment]
> I’ve tried to reset my password but HA comes up with invalid authentication. I tried the token method and it failed to initialise Life360 in HA. 
> 
> any suggestions?

Ignore my last, tried it again and it worked. Fully functioning again 😊

**Comment by vectorr66** [comment]
Does it mess things up when we get our token on the desktop and then logout and log back in on our phone?

**Comment by RoulstoneR** [comment]
Anybody having this issue now? Just done a new install of HA and Life360 plugin. Wont accept username and password, I get an "Invalid authentication" error.

Can't get the token method to work either.

My phone number is not verified and the username and password are correct as they work fine in the app.

**Comment by RoulstoneR** [comment]
This is what's in the log

![Image](https://github.com/user-attachments/assets/6d0d1b20-6c4e-4c98-bff5-112ac6717383)

**Comment by RoulstoneR** [comment]
Got it working. Had to used the token method with email.

**Comment by pnbruckner** [comment]
@RoulstoneR, did you read: https://github.com/pnbruckner/ha-life360#note-on-updating-circles--members-lists

**Comment by RoulstoneR** [comment]
@pnbruckner yeah I did. On further investigation it looks like the app no longer askes for the password. You now suppy the email and you get a pass code sent to email.

I tried several times during the day.

Eventually got the token method to work the following day.

Thanks for the response.

**Comment by fredamn76** [comment]
I manage to authenticate and the sensor Life360 online  seems to be online. However no other entities shows up yet.

**Comment by fredamn76** [comment]
> I manage to authenticate and the sensor Life360 online seems to be online. However no other entities shows up yet.

Still no user entities after 12 hours

**Comment by fredamn76** [comment]
> [@fredamn76](https://github.com/fredamn76), what version are you using? What did you set "Max GPS accuracy"/"GPS accuracy radius limit" to? Do you see any WARNING or ERROR messages in the log?

I have tried both 0.5.4 and 0.6.0b0. Same result.

 "Max GPS accuracy"/"GPS accuracy radius limit" is set to 50 m

I get this error in the log: Could not retrieve full Circles & Members list from server; will retry

**Comment by pnbruckner** [comment]
@fredamn76 please see: https://github.com/pnbruckner/ha-life360#note-on-updating-circles--members-lists

**Comment by fredamn76** [comment]
> > "Max GPS accuracy"/"GPS accuracy radius limit" is set to 50 m
> 
> Also, until you get reasonable results, you should either make this number much larger or leave the limit empty (i.e., no limit.)

I have changed it to 150 meters.
